### ISSUE TITLE
Example Error - port and user switched

### ISSUE THREAD

**Issue by domoran** [blacknon/go-sshlib#23]
On the example in the README.md file the port and user arguments of CreateClient are switched ... This yields a non obvious error getAddrInfoW: class not found ... It cost me some time to figure out the reason. 

```
    // Connect ssh server
    err := con.CreateClient(host, user, port, []ssh.AuthMethod{authMethod})
    if err != nil {
        fmt.Println(err)
        os.Exit(1)
    }
```

Please correct the example in the readme too ;-)

**Comment by domoran** [comment]
It is also mixed up on the sshlib docs: https://pkg.go.dev/github.com/blacknon/go-sshlib@v0.1.8#section-readme

**Comment by blacknon** [comment]
Oh, I just noticed.
Thank you, I'll fix it!

**Comment by blacknon** [comment]
Addressed at pr #24 .
Thanks!
